frontend/test/__support__/ui.tsx

```diff
@@ -59,14 +58,30 @@ export function renderWithProviders(
     initialState = _.pick(initialState, ...publicReducerNames) as State;
   }
 
+  const history = withRouter
+    ? createMemoryHistory({ entries: [initialRoute] })
+    : undefined;
+
   const reducers = mode === "default" ? mainReducers : publicReducers;
-  const store = getStore(reducers, initialState);
+
+  if (withRouter) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    reducers.routing = routerReducer;
```

Comment:
I believed you could use this to circumvent ts complaining. Then the comments could be removed.

```suggestion
    Object.assign(reducers, { routing: routerReducer });
```